### ISSUE TITLE
about diffusers inference

### ISSUE THREAD

**Issue by zy0406** [Lightricks/LTX-Video-Trainer#30]
I trained a lora, but the inference result is noisy; then I tested the inference video of the original model and got the noisy video too；
model_source = "LTXV_13B_097_DEV"
diffusers                 0.34.0.dev0 

`model_source = "LTXV_13B_097_DEV"
components = load_ltxv_components(
    model_source=model_source,
    load_text_encoder_in_8bit=False,
    transformer_dtype=torch.bfloat16,
    vae_dtype=torch.bfloat16,
)
pipe = LTXPipeline(
    scheduler=deepcopy(components.scheduler),
    vae=components.vae,
    text_encoder=components.text_encoder,
    tokenizer=components.tokenizer,
    transformer=components.transformer,
)
pipe.to(device)

def generate_video(text_prompt, output_path="out.mp4"):
  gen = torch.Generator(device=device).manual_seed(42)
  with torch.no_grad(), autocast(device_type=device, dtype=torch.bfloat16):
      result = pipe(
          prompt=text_prompt,
          negative_prompt=negative_prompt,
          width=768,
          height=480,
          num_frames=57,
          num_inference_steps=50,
          generator=gen,
          max_sequence_length=256,
          
      )
  video = result.frames[0]
  export_to_video(video, output_path, fps=25)
  return output_path

generate_video("A cat playing with a ball", "output_t.mp4")`
